src/app/public/modules/split-view/split-view.component.ts

```diff
@@ -193,10 +191,6 @@ export class SkySplitViewComponent implements OnInit, OnDestroy {
       .subscribe((message: SkySplitViewMessage) => {
         this.handleIncomingMessages(message);
       });
-
-    this.mediaQuerySubscription = this.mediaQueryService.subscribe(breakpoint => {
-      this.coreAdapterService.setResponsiveContainerClass(this.elementRef, breakpoint);
-    });
```

Comment:
☝️ The root of the issue.
We were adding an unnecessary responsive class to the host element (`<sky-split-view>`). However, we are also adding a responsive class to the workspace. Child components that get put into the workspace, like tiles, have class declarations are looking for `:host(.sky-responsive-lg/md/sm/etc...)`. Because there were two different parent elements with these responsive classes, it would always revert to the highest element, causing a rendering issue.

This might shine a light on a larger issue with our responsive mixins, but since this particular class wasn't needed I just removed it to solve this immediate problem.